packages/generator/src/sdk-metadata/pregenerated-lib.ts

```diff
@@ -13,6 +10,8 @@ export function getServiceDescription(
   options: GeneratorOptions
 ): string {
   return options.s4hanaCloud
-    ? s4hanaCloudDescription(service.directoryName)
-    : genericDescription(service.directoryName);
+    ? packageDescription(service.directoryName, (directoryNameRaw: string) =>
```

Comment:
[q] @marikaner could we perhaps remove the s4hanaCloud case completly and just use the name as is? In the future customers will create their own services anyhow and they will not really care what is in the package.json descirption field of the generated service I guess.

Reply:
I agree. Please see my other comment regarding the `packageDescription` function. I would just use what is in the service mapping / options per service (and if there is nothing this should be filled with a default initially).

Reply:
removed `s4hanaCloud` case

Reply:
the service name that is in the service mapping/options per service comes from [`VdmServiceMetadata.speakingModuleName`](https://github.com/SAP/cloud-sdk-js/blob/8678c543f5193df838d4b440f232598b2a911a6f/packages/generator/src/service/readme.ts#L15). I use this for all the service names.